examples/step-71/doc/intro.dox

```diff
@@ -0,0 +1,461 @@
+  good enough to guarantee that $u\in H^2$ in this case (smoothly
+  bounded, convex domains would result in $u\in H^3$, but we don't
+  need this much regularity). On the other hand, if the domain is not
+  convex, we can not guarantee that the solution of the mixed system
+  is in $H^2$, and consequently may obtain a solution that can't be
+  equal to the solution of the original biharmonic equation.
+
+The more complicated situation is if we have the "clamped" boundary
+conditions, i.e., if the equation looks like this:
+@f{align*}{
+  \Delta^2 u(\mathbf x) &= f(\mathbf x)
+  \qquad \qquad &&\forall \mathbf x \in \Omega, \\
+  u(\mathbf x) &= g(\mathbf x) \qquad \qquad
+  &&\forall \mathbf x \in \partial\Omega, \\
+  \frac{\partial u(\mathbf x)}{\partial \mathbf n} &= j(\mathbf x) \qquad \qquad
+  &&\forall \mathbf x \in \partial\Omega.
+@f}
+The same trick with the mixed system does not work here, because we
+would end up with <i>both</i> Dirichlet and Neumann boundary conditions for
+$u$, but none for $v$.
+
+
+The solution to this conundrum arrived with the Discontinuous Galerkin
+method wave in the 1990s and early 2000s: In much the same way as one
+can use <i>discontinuous</i> shape functions for the Laplace equation
+by penalizing the size of the discontinuity to obtain a scheme for an
+equation that has one derivative on each shape function, we can use a
+scheme that uses <i>continuous</i> (but not $C^1$ continuous) shape
+functions and penalize the jump in the derivative to obtain a scheme
+for an equation that has two derivatives on each shape function. In
+analogy to the Interior Penalty (IP) method for the Laplace equation,
+this scheme for the biharmonic equation is typically called the $C^0$ IP
+(or C0IP) method, since it uses $C^0$ (continuous but not continuously
+differentiable) shape functions with an interior penalty.
+
+
+<h3> Derivation of the $C^0$ IP method </h3>
+
+We base this program on the $C^0$ IP method presented by Susanne
+Brenner and Li-Yeng Sung in the paper "C$^0$ Interior Penalty Method
+for Linear Fourth Order Boundary Value Problems on polygonal
+domains'' @cite Brenner2005 , where the method is
+derived for the biharmonic equation with "clamped" boundary
+conditions.
+
+As mentioned, this method relies on the use of $C^0$ Lagrange finite
+elements where the $C^1$ continuity requirement is relaxed and has
+been replaced with interior penalty techniques. To derive this method,
+we consider a $C^0$ shape function $v_h$ which vanishes on
+$\partial\Omega$. Since the higher order derivatives of $v_h$ have two
+values on each interface $e\in \mathbb{F}$ (shared by the two cells
+$K_{+},K_{-} \in \mathbb{T}$), we cope with this discontinuity by
+defining the following single-valued functions on $e$:
+@f{align*}{
+\Bigg[\!\Bigg[\frac{\partial^k v_h}{\partial \mathbf n^k}\Bigg]\!\Bigg]&= \frac{\partial^k v_h|_{K_+}}{\partial \mathbf n^k} \bigg |_e - \frac{\partial^k v_h|_{K_-}}{\partial \mathbf n^k} \bigg |_e,\\
```

Comment:
we use [u] and {u} in FEInterfaceValues, see https://www.dealii.org/developer/doxygen/deal.II/classFEInterfaceValues.html#a7827502cb506dbd5d6b2f740184f0a21

do you mind switching to that notation?